azurerm/internal/services/policy/policy_assignment_resource.go

```diff
@@ -113,6 +113,16 @@ func resourceArmPolicyAssignment() *schema.Resource {
 				DiffSuppressFunc: structure.SuppressJsonDiff,
 			},
 
+			"enforcement_mode": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
```

Comment:
You mentioned that this attribute defaults to `Default` and we can make that change in the schema by adding `Default: string(policy.Default),`

Reply:
I declare a default, however since the type is now boolean I simply put it to 'true'